lemur/plugins/lemur_linuxdst/remote_host.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+from lemur.certificates import service
+import paramiko
+import stat
+
+
+def copy_cert(cert_cn, dst_user, dst_priv, dst_priv_key, dst_host, dst_port, dst_dir, dst_file, dst_data):
+
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    if dst_priv_key is None:
+        priv_key = paramiko.RSAKey.from_private_key_file(dst_priv)
+    else:
+        priv_key = paramiko.RSAKey.from_private_key_file(dst_priv, dst_priv_key)
+    ssh.connect(dst_host, username=dst_user, port=dst_port, pkey=priv_key)
+    sftp = ssh.open_sftp()
+    try:
+        sftp.mkdir(dst_dir)
+    except IOError:
```

Comment:
The assumption is that the directory doesn't exist and it will be created. So far I have tested two cases:
1) the directory doesn't exist yet
2) the directory exists because it was previously created from the plugin